presto-elasticsearch/src/test/java/com/facebook/presto/elasticsearch/EmbeddedElasticsearchNode.java

```diff
@@ -0,0 +1,91 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.facebook.presto.elasticsearch;
+
+import com.google.common.collect.ImmutableList;
+import org.elasticsearch.client.Client;
+import org.elasticsearch.common.settings.Settings;
+import org.elasticsearch.transport.Netty4Plugin;
+
+import java.io.Closeable;
+import java.io.File;
+import java.io.IOException;
+import java.io.UncheckedIOException;
+import java.util.concurrent.atomic.AtomicBoolean;
+
+import static com.google.common.io.MoreFiles.deleteRecursively;
+import static com.google.common.io.RecursiveDeleteOption.ALLOW_INSECURE;
+
+public class EmbeddedElasticsearchNode
+        implements Closeable
+{
+    private final File elasticsearchDirectory;
+    private final AtomicBoolean running = new AtomicBoolean();
+
+    private ElasticsearchNode node;
+
+    public static EmbeddedElasticsearchNode createEmbeddedElasticsearchNode()
+    {
+        return new EmbeddedElasticsearchNode();
+    }
+
+    EmbeddedElasticsearchNode()
+    {
+        try {
+            elasticsearchDirectory = File.createTempFile("elasticsearch-test", Long.toString(System.nanoTime()));
```

Comment:
You don't need to give it the current time as a suffix, it already generates a random value and sticks that in into the temp file path. So, you can give it a simple constant suffix too.